Makefile

```diff
@@ -132,6 +132,10 @@ docker-push:
 # Rules for verification, formatting, linting, testing and cleaning #
 #####################################################################
 
+.PHONY: tidy
+tidy:
+	@GO111MODULE=on go mod tidy
```

Comment:
Do we not need
```
	@cd $(LOGCHECK_DIR); go mod tidy
```
as well which was previously there.

Reply:
Fixed.